requirements/core-requirements.yaml

```diff
@@ -41,6 +41,11 @@ sqlalchemy:
   minimum: "1.4.0"
   max_major_version: 1
 
+sqlparse:
+  pip_release: sqlparse
+  minimum: "0.4.0"
+  max_major_version: 0
```

Comment:
sqlparse is already present in `skinny-requirements`. Shall we bump the minimum version to `0.4.0` there?

Reply:
https://github.com/mlflow/mlflow/blob/bfbef7bfb9901da279b6ed7929831d02c4070fc4/requirements/skinny-requirements.yaml#L62-L66

Reply:
addressed.